src/main/java/org/skriptlang/skript/elements/expressions/displays/ExprDisplayHeightWidth.java

```diff
@@ -0,0 +1,126 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.elements.expressions.displays;
+
+import org.bukkit.entity.Display;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Display Height/Width")
+@Description({
+	"Returns or changes the height or width of <a href='classes.html#display'>displays</a>.",
+	"Rendering culling bounding box spans horizontally width/2 from entity position, "+
+	"and the part beyond will be culled.",
+	"If set to 0, no culling on both vertical and horizonal directions. Default is 0.0"
```

Comment:
```suggestion
	"Returns or changes the height or width of <a href='classes.html#display'>displays</a>.",
	"The rendering culling bounding box spans horizontally width/2 from entity position, "+
	"and the part beyond will be culled.",
	"If set to 0, no culling will occur on both the vertical and horizontal directions. Default is 0.0."
```
I'm not sure how to interpret the 2nd sentence. Does it go from 1/4 to 3/4 along the shape? Is it half the shape on both sides (doubt that, it'd just be the same as the normal box). Is it only dependent on width, or does height matter? I think a little clarification would help :).